netci.groovy

```diff
@@ -70,6 +77,15 @@ set DOTNET_CLI_UI_LANGUAGE=es
             buildCommand = "./build.sh --linux-portable --skip-prereqs --configuration ${configuration} --targets Default"
         }
     }
+    else if (os == 'Linux_NoSuffix') {
+        osUsedForMachineAffinity = 'Ubuntu16.04';
+        if ((architecture == 'arm') || (architecture == 'arm64')) {
+            buildCommand = "./build.sh --linux-portable --skip-prereqs --architecture ${architecture} --configuration ${configuration} --targets Default /p:CLIBUILD_SKIP_TESTS=true /p:DropSuffix=true"
```

Comment:
Is it possible to avoid repeating the whole command line here?  Instead, for the NoSuffix builds, could we just append `/p:DropSuffix=true` to the existing command lines?